packages/lesswrong/server/rateLimitUtils.ts

```diff
@@ -205,22 +210,51 @@ export async function rateLimitDateWhenUserNextAbleToPost(user: DbUser): Promise
   // fetch the posts from within the maxTimeframe
   const postsInTimeframe = await getPostsInTimeframe(user, maxHours);
 
-  const rateLimitInfos = getPostRateLimitInfos(user, postsInTimeframe, modRateLimitHours, manualPostRateLimit, recentKarmaInfo);
+  const features = await getFeatures(user)
```

Comment:
Is there any reason this can't be part of the `Promise.all`?   (Also I continue to think that names like `features` and `getFeatures` are almost maximally uninformative)